src/domain/octree/OctreePacketProcessor.ts

```diff
@@ -72,36 +74,57 @@ class OctreePacketProcessor {
 
     /*@devdoc
      *  Triggered when an entity data packet is received.
-     *  @function OctreePacketProcessor.addedEntity
+     *  @callback OctreePacketProcessor.entityData
+     *  @param {EntityDataDetails[]} entityData - The entity data information for one or more entities.
      *  @returns {Signal}
      */
-    get addedEntity(): Signal {
-        return this.#_addedEntity.signal();
+    get entityData(): Signal {
+        return this.#_entityData.signal();
     }
 
     // Listener
-    // eslint-disable-next-line
-    // @ts-ignore
-    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {// eslint-disable-line
+    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {
         // C++ void OctreePacketProcessor::processPacket(QSharedPointer<ReceivedMessage> message, SharedNodePointer sendingNode)
 
-        const packetType = message.getType();
+        assert(sendingNode !== null, "OctreePacketProcessor.processPacket()", "Sending node is missing.");
 
-        if (packetType === PacketType.OctreeStats) {
+        let messageLocal = message;
+
+        const octreePacketType = messageLocal.getType();
+
+        if (octreePacketType === PacketType.OctreeStats) {
             if (!this.#_haveWarnedOctreeStats) {
                 console.warn("OctreePacketProcessor: OctreeStats packet not processed.");
                 this.#_haveWarnedOctreeStats = true;
             }
-            // WEBRTC TODO: Address further C++ code.
-            return;
+
+            // WEBRTC TODO: Address further C++ code - process OctreeStats packet.
+
+            // The stats message is always 222 bytes long.
+            const statsMessageLength = 222;
+            // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+            const piggybackBytes = messageLocal.getMessage().byteLength - statsMessageLength;
```

Comment:
```suggestion
            const STATS_MESSAGE_LENGTH = 222;
            const piggybackBytes = messageLocal.getMessage().byteLength - STATS_MESSAGE_LENGTH;
```
Constants upper-case per C++ coding standard.